dbbackup/management/commands/dbbackup.py

```diff
@@ -60,6 +60,13 @@ class Command(BaseDbBackupCommand):
         make_option(
             "-x", "--exclude-tables", default=None, help="Exclude tables from backup"
         ),
+        make_option(
+            "-n",
+            "--schema",
+            action="append",
+            default=[],
+            help="Specify schema(s) to backup. Can be used multiple times.",
```

Comment:
I have a general question, is there an actual use-case of using multiple schemas for one backup session?

Reply:
I haven't encountered before but Postgres supports it.